sdks/python/apache_beam/runners/worker/operations.py

```diff
@@ -286,6 +287,10 @@ def _read_side_inputs(self, tags_and_types):
     # provided directly.
     assert self.side_input_maps is None
 
+    # Get experiments active in the worker to check for side input metrics exp.
+    experiments = set(RuntimeValueProvider(
```

Comment:
<!--thread_id:cc_163728724_t; commit:b68f9f26428de8ed2f34c0919ee91a1662402d86; resolved:1-->
<!--section:context-quote-->
> **pabloem** wrote:
> I don't believe that instance creation represents a serious performance cost here. This is only incurred once per side input-consuming operation in a map task. I'll add it if you think it's a must.

<!--section:body-->
I was thinking more from a code style perspective of this repeated code. Maybe you can create a JIRA to improve this.